cypress/integration/place-an-order.js

```diff
@@ -0,0 +1,61 @@
+/* eslint no-undef: 0 */
+
+describe('shopping path', () => {
+  beforeEach(() => {
+    cy.visit('/p/38143c0c-c9b0-448c-93cd-60eb90d8da57/aspesi-shirt-h805-white').wait(2000);
+    cy.url().should('include', '/p/38143c0c-c9b0-448c-93cd-60eb90d8da57/aspesi-shirt-h805-white');
+    cy.get('[data-cy=product-cart_add]').click().wait(500);
+    cy.get('[aria-label="cart"]').click();
+    cy.get('[data-cy=cart-sidebar-btn_checkout]').click({ force: true });
+    cy.scrollTo('top');
+  });
+  afterEach(() => {
+    cy.clearLocalStorage();
+    cy.clearCookies();
+    cy.reload(true);
+  });
+  it('Place an order as a guest', () => {
+    const firstName = cy.get('[data-cy=personal-details-input_firstName]');
+    const lastName = cy.get('[data-cy=personal-details-input_lastName]');
+    firstName.clear().type('John');
+    lastName.clear().type('Doe');
+    cy.get('[data-cy=personal-details-input_email]').clear().type('john@doe.com');
+    cy.get('[data-cy=personal-details-btn_continue]').click();
+    cy.wait(500);
+
+    cy.url().should('include', '/shipping');
+    cy.get('[data-cy=shipping-details-input_firstName]').type('John');
+    cy.get('[data-cy=shipping-details-input_lastName]').type('Doe');
+    cy.get('[data-cy=shipping-details-input_streetName]').type('Dmowskiego');
+    cy.get('[data-cy=shipping-details-input_apartmanet]').type('5');
+    cy.get('[data-cy=shipping-details-input_city]').type('Wrocław');
+    cy.get('[data-cy=shipping-details-input_postalCode]').type('50-500');
+    cy.get('[data-cy=shipping-details-input_state]').type('Dolnośląskie');
+    cy.get('[data-cy=shipping-details-input_phone]').type('666-666-666');
+    cy.get('[data-cy=shipping-details-select_country]').click().wait(500);
+    cy.get('.sf-select__options > :nth-child(2)').click();
+    cy.get('ul.sf-select__options').children().should('have.length', 4);
+    cy.wrap(['United States', 'United Kingdom', 'Italy', 'Poland']).its(0).should('eq', 'United States');
```

Comment:
same as above, wdyt about changing it to
```
cy.get('ul.sf-select__options').children().should((countryFormOptions) => {
  expect(countryFormOptions).to.have.length(4)
  expect(countryFormOptions.eq(0)).to.contain('United States')
})
```

Reply:
I think also that it is better to add `[data-cy=shipping-details-select_country]` to selector. So it will be `'[data-cy=shipping-details-select_country] .sf-select__options'`. If there will be another select on the page then it can break

Reply:
ok, thanks @gibkigonzo !